python/JetSelector_cfi.py

```diff
@@ -21,7 +21,7 @@
     jerShift                  = cms.int32(1), 
     jetPtMin                  = cms.double(30),
     jetPtMax                  = cms.double(1000000),
-    jetAbsEtaMax              = cms.double(5.0),
+    jetAbsEtaMax              = cms.double(2.4),# 5.0
     jetCSVDiscMin             = cms.double(-10000),
```

Comment:
@tmitchel Please, the idea is that no object cfi file should be changed (these are used for other analyses as well.) These kinds of changes should be done in the OS2LAna_cfi or better, in the os2lana_cfg.py